rules/order/tests/StmtOrderTest.php

```diff
@@ -80,7 +87,19 @@ public function testReorderClassStmtsByOldToNewKeys(): void
         $class = $this->getTestClassNode();
         $actualClass = $this->stmtOrder->reorderClassStmtsByOldToNewKeys($class, self::OLD_TO_NEW_KEYS);
         $expectedClass = $this->getExpectedClassNode();
-        $this->assertSame($expectedClass->stmts, $actualClass->stmts);
```

Comment:
I think this is good enoug. Checking node subproperties will only make test complicated

Reply:
Agree

Reply:
I mean to keep this:

```php
$this->assertSame($expectedClass->stmts, $actualClass->stmts);
```

And remove the name node comparing

Reply:
Sorry, misunderstood what you meant. It doesn't work because objects inside the `stmts` are not the same, they are just equal :D
Even though they are the instances of the same class they are different objects and reference different locations in memory. (I don't understand it into detail so maybe, I'm writting bullshit :D)
```php
$this->assertSame(new \stdClass(), new \stdClass());
```
For example, the assert above will fail.

Reply:
If we'll decide to leave it as it is now we can get rid of `$expectedClass` and use plain string names in assertion, eg:
```php
        $this->assertSame(
            'price',
            $this->nodeNameResolver->getName($actualClass->stmts[1])
        );
```

Reply:
I'm sorry, I didn't get how this test works and I'm used to strict testing, because if it was not strict if failed many months later and shoot me in the head :D

Let's work with current version